packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -172,25 +187,73 @@ class BottomNavigationBar extends StatefulWidget {
   /// The index into [items] of the current active item.
   final int currentIndex;
 
+  /// The z-coordinate of this [BottomNavigationBar].
```

Comment:
It's effectively the 'shadow'. Do we have any example of a `.elevation` where we mention shadow that you could steal?

Reply:
Well I have a macro pointing to the `Material.elevation`'s dart doc which has the line:
`/// This controls the size of the shadow below the material.`

Do you think that is sufficient?